lorien/Misc/Settings.gd

```diff
@@ -63,59 +63,75 @@ func set_value(key: String, value = null):
 	_save_settings()
 	
 # -------------------------------------------------------------------------------------------------
-func _set_default_shortcuts():	
+func _set_default_shortcuts():
 	for action_name in InputMap.get_actions():
+		if _config_file.has_section_key("shortcuts", action_name):
+			continue
+			
+		var shortcuts = []
 		for event in InputMap.get_action_list(action_name): 
-			if typeof(event) == TYPE_OBJECT && event.is_class("InputEventJoypadButton"):
-				continue
-			if _config_file.has_section_key("shortcuts", action_name):
-				continue
-			var shortcut = OS.get_scancode_string(event.get_scancode_with_modifiers())
-			_config_file.set_value("shortcuts", action_name, shortcut)
-			continue # for now every action can have only one shortcut
-			# parsing multiple shortcuts should be ease -  this is just a concept
+			if event is InputEventKey:
+				var shortcut = OS.get_scancode_string(event.get_scancode_with_modifiers())
+				shortcuts.append("KB:" + shortcut)
+			elif event is InputEventJoypadButton:
+				var shortcut = Input.get_joy_button_string(event.button_index)
+				shortcuts.append("JOY:" + shortcut)
+			else:
+				printerr("[DEV] unimplemented input event type: ", event.get_class())
```

Comment:
[print_debug](https://docs.godotengine.org/en/stable/classes/class_@gdscript.html?highlight=print_debug#class-gdscript-method-print-debug) maybe is a better fit here?